dallinger/recruiters.py

```diff
@@ -727,6 +736,52 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(
+        self,
+        assignment_id: str,
+        participants: list[Participant],
+        reward: float,
+        wage_per_hour: float,
+    ):
+        if self.screen_out_allowed(assignment_id, participants, reward, wage_per_hour):
+            try:
+                return self.prolificservice.screen_out(
+                    study_id=self.current_study_id,
+                    submission_id=assignment_id,
+                    bonus_per_submission=reward,
+                    increase_places=self.config.get("auto_recruit"),
+                )
+            except ProlificServiceException as ex:
+                logger.exception(str(ex))
+
+    def screen_out_allowed(
+        self,
+        assignment_id: str,
+        participants: list[Participant],
+        reward: float,
+        wage_per_hour: float,
+    ):
+        if self.prolificservice.get_study(assignment_id)["is_custom_screening"]:
+            raise ProlificRecruiterException(
+                f"Prolific study (ID {self.current_study_id}) doesn't allow screening-out of participants"
+            )
+
+        # Minimum wage thresholds: https://researcher-help.prolific.com/en/article/2273bd
```

Comment:
Or would it rather be £11.44/hour for UK  based on the table here:
https://researcher-help.prolific.com/en/article/2273bd ?

Reply:
@pmcharrison Can you have a look?

Reply:
6 is correct, I believe.